src/Tablet/Sensors/SensorsTab.js

```diff
@@ -99,28 +99,22 @@ const SensorsTab = () => {
 
   const renderCard = type => {
     return (
-      <IonCol className={s.column}>
-        <IonCard color="dark">
-          <IonCardHeader color="medium">{type.toUpperCase()}</IonCardHeader>
-          <IonCardContent>{renderType(type)}</IonCardContent>
-        </IonCard>
-      </IonCol>
+      <IonCard color="dark">
+        <IonCardHeader color="medium">{type.toUpperCase()}</IonCardHeader>
+        <IonCardContent>{renderType(type)}</IonCardContent>
+      </IonCard>
     );
   };
 
   let col = 0;
   return (
-    <div style={{ padding: 20, marginTop: 10 }}>
-      <IonRow>
-        {renderCard(types[col++])}
-        {renderCard(types[col++])}
-        {renderCard(types[col++])}
-      </IonRow>
-      <IonRow>
-        {renderCard(types[col++])}
-        {renderCard(types[col++])}
-        {renderCard(types[col++])}
-      </IonRow>
+    <div className={s.gridContainer}>
+      <div className={s.gridItem}>{renderCard(types[col++])}</div>
```

Comment:
`div.gridItem` looks like an overkill. Removing this element altogether should do
![image](https://user-images.githubusercontent.com/733074/61361662-1cdcf100-a881-11e9-9b81-bf366118ca23.png)

Reply:
In that casse all you need is this simple loop
```jsx
<div className={s.gridContainer}>
  {types.map(renderCard)}
</div>
```

Reply:
cant believe i didnt recognize this!